mula/scheduler/schedulers/normalizer.py

```diff
@@ -318,7 +306,7 @@ def is_task_allowed_to_run(self, normalizer: Plugin) -> bool:
         Returns:
             True if the task is allowed to run, False otherwise.
         """
-        if not normalizer.enabled:
+        if normalizer.enabled is False:
```

Comment:
The previous implementation is the more Pythonic way of dealing with this. The current suggested implementation is generally unnecessary unless you need to differentiate between `False` and other falsy values (e.g., `None`, `0`, `''`, etc.).